alma/controllers/admin/AdminAlmaRefunds.php

```diff
@@ -80,7 +80,7 @@ public function ajaxProcessRefund()
                 }
                 break;
             case 'total':
-                $isTotal = true;
```

Comment:
then this variable is never useful :) ( it's always false )

 you could simply do :
```
            $refundResult = $this->runRefund($paymentId, $amount, false);
```

and remove this variable in this function